packages/babel-helper-create-class-features-plugin/src/decorators.ts

```diff
@@ -943,14 +1055,6 @@ function transformClass(
   }
 
   if (!classDecorators && !hasElementDecorators) {
-    // If nothing is decorated but we have assignments, it must be the memoised
-    // computed keys of class accessors
-    if (assignments.length > 0) {
-      path.insertBefore(assignments.map(expr => t.expressionStatement(expr)));
-
-      // Recrawl the scope to make sure new identifiers are properly synced
-      path.scope.crawl();
-    }
```

Comment:
This branch is removed because the computed key in a vanilla class accessor is now memoised in place.